packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,654 @@
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Deactivate Authenticator
+
+<ApiOperation method="post" url="/api/v1/authenticators/{authenticatorId}/lifecycle/deactivate" />
+
+#### Request Path Parameters
+
+| Parameter          | Type   | Description                                            |
+| ------------------ | ------ | ------------------------------------------------------ |
+| `authenticatorId`  | String | The Authenticator's unique identifier.                 |
+
+#### Response Body
+
+HTTP 200:
+[Authenticator Object](#authenticator-object)
+
+#### Example Usage
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate"
+```
+
+##### Response
+
+```json
+{
+    "type": "security_key",
+    "id": "aut1nd8PQhGcQtSxB0g4",
+    "key": "webauthn",
+    "status": "INACTIVE",
+    "name": "Security Key or Biometric",
+    "created": "2020-07-26T21:16:37.000Z",
+    "lastUpdated": "2020-07-26T22:01:46.000Z",
+    "_links": {
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "methods": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/methods",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "activate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/activate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Authenticator Admin API Objects
+
+This API has one object:
+* Authenticator Object
+
+### Authenticator Object
+
+#### Authenticator Properties
+
+The Authenticator model defines the following attributes
+
+| Property      | Type                                                            | Description                                                                     |
+| ------------- | --------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| `_links`      | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this object                                                  |
```

Comment:
nit: not sure if the pattern is to end everything with a period vs not, but if we're ending descriptions with a period, then we could add to this description.